docs/errors.md

```diff
@@ -0,0 +1,29 @@
+## Errors
+
+`pg_hint_plan` stops parsing on any error and uses hints already parsed on the
+most cases. Following are the typical errors.
+
+#### Syntax errors
+
+Any syntactical errors or wrong hint names are reported as a syntax error.
+These errors are reported in the server log with the message level specified by
+`pg_hint_plan.message_level` if `pg_hint_plan.debug_print` is on and above.
+
+#### Object misspecifications
+
+Object misspecifications result in silent ignorance of the hints. This kind of
+error is reported as "not used hints" in the server log by the same condition
+as syntax errors.
+
+#### Redundant or conflicting hints
+
+The last hint will be active when redundant hints or hints conflicting with
```

Comment:
In the event of a redundant or conflicting hint, the last hint will be the one considered.